index.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+//connect to database
+$db = new PDO('mysql:host=db; dbname=tim-collection', 'root', 'password');
+
+//prepare statement to fetch player information and ratings from database
+$query = $db->prepare("SELECT `id`,`title`, `author`, `covertype`, `published`, `condition`, `signed`,`image` FROM `first-editions`;");
+
+//set fetch attribute to create resulting fetch as an associative array
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
I think this line needs to go directly under where you connect to the database (line 4) as you are still getting duplicates in the array.

Reply:
Thank you. Order of DB query commands amended to remove duplication.